src/js/apps/patients/patient/archive/archive_app.js

```diff
@@ -13,7 +11,9 @@ export default App.extend({
     this.getRegion('content').startPreloader();
   },
   beforeStart({ patient }) {
-    const filter = { status: STATE_STATUS.DONE };
+    const currentOrg = Radio.request('bootstrap', 'currentOrg');
+    const states = currentOrg.getStates();
+    const filter = { status: states.filterByDone(true).pluck('status').join(',') };
```

Comment:
Ah so this should be `state: states.filterBy...pluck('id').join...`. We don't want to filter by status any longer.

Reply:
Oops missed that...